lib/include/pl/core/errors/error.hpp

```diff
@@ -16,14 +15,14 @@ namespace pl::core::err {
         std::string formatRuntimeError(
                 const Location& location,
                 char prefix,
-                const std::string &title,
-                const std::string &description,
-                const std::string &hint);
+                const std::string& title,
```

Comment:
This was fine as it was before.
I only put the `&` and `*` towards the type if it applies to the type and not the variable.
So it goes to the left on return types, template parameters or using declarations but on the right for variable declarations and parameters